src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -753,13 +878,50 @@ else if (data.getType() == RmdChunkOutputFinishedEvent.TYPE_INTERACTIVE &&
             // set dirty state if necessary
             setDirtyState();
          }
+
+         if (satelliteChunks_.contains(data.getChunkId()))
+         {
+            forwardEventToSatelliteChunk(
+               docUpdateSentinel_.getId(),
+               data.getChunkId(),
+               new ChunkSatelliteOutputFinishedEvent(
+                  ensureVisible,
+                  data.getScope()
+               )
+            );
+         }
       }
    }
 
+   @Override
+   public void onConsoleWriteOutput(ConsoleWriteOutputEvent event)
```

Comment:
Yes, every satellite has its own `ClientEventDispatcher` (https://github.com/rstudio/rstudio/blob/8b8d6c4d0040b46c8fe23d6fa02bc2fbf1cf6a29/src/gwt/src/org/rstudio/studio/client/common/satellite/Satellite.java#L58) which is the only place that the `ConsoleWriteOutputEvent` is fired. Don't sweat it, though; if you have to manually forward to get the behavior you need that's okay (maybe we can take a look at why later this week).